Sources/TripKit/categories/NSNumber+Formatter.swift

```diff
@@ -19,6 +19,20 @@ extension NSNumber {
     return formatter
   }()
   
+  // TODO: Better name / distinction from toMoneyString
```

Comment:
Can you add a comment why we have this as an alternative to `toMoneyString`? Would it maybe make more sense to add a parameter to that existing method to toggle the different behaviours? Seems like the only differences are the fraction digits.

Reply:
was initially thinking of adding parameter to include decimals and let 0 decimal places be the default. But I thought of making a new one for the purpose of having new method caller when it comes to displaying of detailed prices / currency in single unadjustable format (2 decimal places and currency sign) to give devs context of the format used.

Can have the parameter sending within the extension and have it called from functions with context.